app/content/profile/index.js

```diff
@@ -189,6 +388,77 @@ const styles = StyleSheet.create({
     textAlign: "center",
     color: "gray",
   },
+  modalContent: {
```

Comment:
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/82061589/3e8433b1-473b-424f-b888-7e02d046b502)

I think this might actually be scrollViewContent styling, not sure. Could use a bit more bottom padding (tested on Android emulator)